apps/fluent-tester/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui-react-native/tester",
-  "version": "0.90.9",
+  "version": "0.90.7",
```

Comment:
The version is being downgraded, we should put it back to what it was.

Same with apple-theme below.